erpnext/hr/doctype/employee/employee.json

```diff
@@ -843,12 +844,18 @@
    "fieldname": "twitter",
    "fieldtype": "Data",
    "label": "Twitter"
+  },
+  {
+   "fieldname": "email_id_of_reports_to",
+   "fieldtype": "Data",
```

Comment:
@pritishsal It should be Read only field

Reply:
This was not mentioned in Dod. I Will update accordingly and commit.